modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/VerticalBar/CustomVerticalBar.scss

```diff
@@ -27,9 +27,3 @@
 		transition: transform 0.5s;
 	}
 }
-
-.dropdown-menu {
-	max-width: 17rem;
-	width: 17rem;
-	z-index: 1035;
-}
```

Comment:
Hey @tomibiro, thank you very much for this contribution.

Indeed, our styles for dropdown uses were very comprehensive, thank you for noticing that.
I'm just closing this PR and opening a [new one](https://github.com/liferay-objects/liferay-portal/pull/2897), adding the necessary styles for using dropdown on objects.